sdk/src/utils/spl-token-utils.ts

```diff
@@ -19,3 +28,61 @@ export function getAssociatedTokenAddressSync(
 
   return address;
 }
+
+// TODO: Move this to common-sdk and use that in this SDK later
+export function wrapSOL(
```

Comment:
yeah i think moving to sync-native will simplify this flow. Let's do it in a dedicated effort though because if we do that we need to verify we are not breaking all the other SDK methods.